### ISSUE TITLE
Always show Test  accuracy 1.0 for run_stacked_autoencoder_supervised.py

### ISSUE THREAD

**Issue by robi56** [blackecho/Deep-Learning-TensorFlow#30]
I just try with the file after a slight modifying for my custom data  **run_stacked_autoencoder_supervised.py**   with doc configuation 

After running the code the output shows  always test accuracy 1.0 . I put some wrong output label in testlabel.txt file , but at this case it also shows accuracy 1.0. 

**The CommandLine Output LookLike** 

> Kingfisher:Deep-Learning-TensorFlow Nandi$ python practice/run_stacked_autoencoder_supervised.py --dae_layers 1024,784,512,256 --dae_batch_size 25 --dae_num_epochs 5 --verbose 1 --dae_corr_type masking --dae_corr_frac 0.0 --finetune_num_epochs 25 --finetune_opt momentum --momentum 0.9 --finetune_learning_rate 0.05 --dae_enc_act_func sigmoid --dae_dec_act_func sigmoid --dae_loss_func cross_entropy --finetune_act_func relu --finetune_loss_func softmax_cross_entropy --dropout 0.7
> Creating /Users/Nazib/.yadlt/stored_models/sdae/ directory to save/restore models
> Creating /Users/Nazib/.yadlt/data/sdae/ directory to save model generated data
> Creating /Users/Nazib/.yadlt/logs/sdae/ directory to save tensorboard logs
> Creating /Users/Nazib/.yadlt/stored_models/sdae/dae-1 directory to save/restore models
> Creating /Users/Nazib/.yadlt/data/sdae/dae-1 directory to save model generated data
> Creating /Users/Nazib/.yadlt/logs/sdae/dae-1 directory to save tensorboard logs
> Creating /Users/Nazib/.yadlt/stored_models/sdae/dae-2 directory to save/restore models
> Creating /Users/Nazib/.yadlt/data/sdae/dae-2 directory to save model generated data
> Creating /Users/Nazib/.yadlt/logs/sdae/dae-2 directory to save tensorboard logs
> Creating /Users/Nazib/.yadlt/stored_models/sdae/dae-3 directory to save/restore models
> Creating /Users/Nazib/.yadlt/data/sdae/dae-3 directory to save model generated data
> Creating /Users/Nazib/.yadlt/logs/sdae/dae-3 directory to save tensorboard logs
> Creating /Users/Nazib/.yadlt/stored_models/sdae/dae-4 directory to save/restore models
> Creating /Users/Nazib/.yadlt/data/sdae/dae-4 directory to save model generated data
> Creating /Users/Nazib/.yadlt/logs/sdae/dae-4 directory to save tensorboard logs
> Training layer 1...
> Tensorboard logs dir for this run is /Users/Nazib/.yadlt/logs/sdae/dae-1/run5
> Reconstruction loss at step 0: -8036.11
> Reconstruction loss at step 1: -9851.44
> Reconstruction loss at step 2: -11081.3
> Reconstruction loss at step 3: -11957.8
> Reconstruction loss at step 4: -12676.6
> Training layer 2...
> Tensorboard logs dir for this run is /Users/Nazib/.yadlt/logs/sdae/dae-2/run5
> Reconstruction loss at step 0: 0.659456
> Reconstruction loss at step 1: 0.658065
> Reconstruction loss at step 2: 0.656675
> Reconstruction loss at step 3: 0.655287
> Reconstruction loss at step 4: 0.653903
> Training layer 3...
> Tensorboard logs dir for this run is /Users/Nazib/.yadlt/logs/sdae/dae-3/run5
> Reconstruction loss at step 0: 0.788582
> Reconstruction loss at step 1: 0.787858
> Reconstruction loss at step 2: 0.787136
> Reconstruction loss at step 3: 0.786414
> Reconstruction loss at step 4: 0.785694
> Training layer 4...
> Tensorboard logs dir for this run is /Users/Nazib/.yadlt/logs/sdae/dae-4/run5
> Reconstruction loss at step 0: 0.743701
> Reconstruction loss at step 1: 0.743354
> Reconstruction loss at step 2: 0.743006
> Reconstruction loss at step 3: 0.742658
> Reconstruction loss at step 4: 0.742312
> Tensorboard logs dir for this run is /Users/Nazib/.yadlt/logs/sdae/run5
> Accuracy at step 0: 1.0
> Accuracy at step 1: 1.0
> Accuracy at step 2: 1.0
> Accuracy at step 3: 1.0
> Accuracy at step 4: 1.0
> Accuracy at step 5: 1.0
> Accuracy at step 6: 1.0
> Accuracy at step 7: 1.0
> Accuracy at step 8: 1.0
> Accuracy at step 9: 1.0
> Accuracy at step 10: 1.0
> Accuracy at step 11: 1.0
> Accuracy at step 12: 1.0
> Accuracy at step 13: 1.0
> Accuracy at step 14: 1.0
> Accuracy at step 15: 1.0
> Accuracy at step 16: 1.0
> Accuracy at step 17: 1.0
> Accuracy at step 18: 1.0
> Accuracy at step 19: 1.0
> Accuracy at step 20: 1.0
> Accuracy at step 21: 1.0
> Accuracy at step 22: 1.0
> Accuracy at step 23: 1.0
> Accuracy at step 24: 1.0
> Test set accuracy: 1.0
> 

**I have create a folder Named Data1  and keep my data** 
[testdata.txt](https://github.com/blackecho/Deep-Learning-TensorFlow/files/494231/testdata.txt)
[testlabel.txt](https://github.com/blackecho/Deep-Learning-TensorFlow/files/494233/testlabel.txt)
[traindata.txt](https://github.com/blackecho/Deep-Learning-TensorFlow/files/494234/traindata.txt)
[trainlabel.txt](https://github.com/blackecho/Deep-Learning-TensorFlow/files/494232/trainlabel.txt)

 **The modified  code I use here** 
[run_stacked_autoencoder_supervised.txt](https://github.com/blackecho/Deep-Learning-TensorFlow/files/494237/run_stacked_autoencoder_supervised.txt)

**Comment by siucaan** [comment]
I meet the same issue while run the "run_rbm.py".
Could anybody give me a hand?
